<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>369</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Export</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,1,1">
     <item row="1" column="2">
      <widget class="QListWidget" name="listWidget_rcv"/>
     </item>
     <item row="1" column="3">
      <widget class="QListWidget" name="listWidget_well"/>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_well">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Well path file</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QListWidget" name="listWidget_mesh"/>
     </item>
     <item row="1" column="1">
      <widget class="QListWidget" name="listWidget_src"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_src">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Source configuration file</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_mesh">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Mesh file</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_rcv">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Receiver configuration file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_frq">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Frequency file: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_frq"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushBtn_frqbrowser">
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listWidget_mesh</tabstop>
  <tabstop>listWidget_src</tabstop>
  <tabstop>listWidget_rcv</tabstop>
  <tabstop>listWidget_well</tabstop>
  <tabstop>lineEdit_frq</tabstop>
  <tabstop>pushBtn_frqbrowser</tabstop>
  <tabstop>pushButton_ok</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
